main.cpp

```diff
@@ -69,6 +70,21 @@ void variableDeclarations()
     
     
     ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int numbers = 5; ignoreUnused(numbers);
+    int ratio = 7;  ignoreUnused(ratio);
+    int fraction = 9; ignoreUnused(fraction);
+    bool truth = 1; ignoreUnused(truth);
+    bool faux = 3; ignoreUnused(faux);
+    bool off = 6; ignoreUnused(off);
+    char name = 8; ignoreUnused(name);
+    char gender = 2; ignoreUnused(gender);
+    char person = 4; ignoreUnused(person);
+    float height = 23; ignoreUnused(height);
+    float length = 25; ignoreUnused(length);
+    float area = 27;ignoreUnused(area);
+    double weight = 29; ignoreUnused(weight);
+    double frequency = 22; ignoreUnused(frequency);
+    double waves = 24; ignoreUnused(waves);
```

Comment:
- [ ] pass them all to a single call of `ignoreUnused()`
- [ ] initialize your `char` variables with single characters, not integers: `'c'`
- [ ] initialize your floats with floating point numbers, not integers.  `3.14f`
- [ ] initialize your booleans with `true/false`, not integers
- [ ] initialize your doubles with doubles, not integers.  `3.14`